db/migrate/20200220164019_add_discipline_to_team_members.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisciplineToTeamMembers < ActiveRecord::Migration[6.0]
```

Comment:
Looks like you're still doing two unnecessary migrations in this commit. You should be able to drop both of them and have the same effect.

Reply:
I'm getting a bit confused with this. On my local I don't have those migration files (I rolled back the DB and deleted manually those 2 migration files), so I'm not sure why they're still there if I pushed my latest version to github.

Reply:
To be clear, this was a comment on the first commit. You did delete the migrations, but it would have been better to have removed them from the commit that introduced them. This is now resolved, but I thought you might find an explanation helpful.